pkg/migration/controllers/clusterpair.go

```diff
@@ -218,10 +218,29 @@ func getClusterPairSchedulerStatus(clusterPairName string, namespace string) (st
 }
 
 func (c *ClusterPairController) cleanup(clusterPair *stork_api.ClusterPair) error {
+	skipDelete := false
+	cpName := ""
```

Comment:
nit: Collect all the clusterPairs and their namespaces so that we can show them in the log message and/or event.